config.txt

```diff
@@ -0,0 +1,9 @@
+ENV_GENETICDIR=/var/genetics
```

Comment:
this doesn't want to be in the repo, we'd keep it in 001_references project with other configs when it is done